src/lib/search-url-builder.ts

```diff
@@ -47,18 +49,27 @@ export const titleFromPath = (all: string[] = []) => {
   const instructor = last(path.split('lessons-by-'))
   const tags = tagsForPath(path)
 
+  const count = config.searchResultCount
+  const humanizedTags = ` ${humanize(tags?.map(toTitleCase))} `
+  const humanizedInstructors = humanize(
+    instructor?.split(`-and-`).map(nameSlugToName),
+  )
+
   if (instructor) {
-    return `${config.searchResultCount}${
-      tags ? ` ${humanize(tags.map(toTitleCase))} ` : ' '
-    }Courses from ${humanize(
-      instructor.split(`-and-`).map(nameSlugToName),
-    )} in ${year}`
-  } else if (tags) {
-    return `${config.searchResultCount}${
-      tags ? ` ${humanize(tags.map(toTitleCase))} ` : ' '
-    }Courses for Web Developers in ${year}`
+    return `${count}${humanizedTags}Courses from ${humanizedInstructors} in ${year}`
   }
-  return `${config.searchResultCount} Courses for Web Developers`
+
+  //TODO: I think we need more tests around tags and years here...
```

Comment:
surprised to not see any added if this is the case! The tests were definitely non-exhaustive.